### ISSUE TITLE
kelew belew

### ISSUE THREAD

**Issue by DenisNosnitsin1970** [BetaMasaheft/Documentation#2512]
Should we have an ID for the mythical ethnic group Kelew Belew?

**Comment by CarstenHoffmannMarburg** [comment]
If it is clear that it is a mythical ethnic group, I would support an ID. It is easier to refer to them and talk about them, as it is the case already with PRS4735Gog and PRS4736Magog and it should also be for Amazons and similar cases.

**Comment by eu-genia** [comment]
Yes, but is it sure it is not Balaw Kalaw (I mean the order of the words)? Balaw Kalaw is (also) a name of an archaeological site in Eritrea

**Comment by DenisNosnitsin1970** [comment]
This is the same, the word order may easily change. Yes, it is a site, called most probably after the group (believed to have been there once).

**Comment by eu-genia** [comment]
has this been done?

**Comment by eu-genia** [comment]
@DenisNosnitsin1970

**Comment by DenisNosnitsin1970** [comment]
Not yet; I will.
